l3build/l3build.lua

```diff
@@ -2069,12 +2070,12 @@ end
 
 -- Unpack the package files using an 'isolated' system: this requires
 -- a copy of the 'basic' DocStrip program, which is used then removed
-function unpack()
+function unpack(sourcedir)
```

Comment:
I'm currently working on properly documenting the internal functions. My feeling at the moment is that the top-level ones should have arguments which are more-or-less restricted to just the command line `<files>` (where appropriate). (There are a few issues with that which is why I'm working through the code!) Here, I'm not sure having an argument makes sense: the 'main' source dir is 'here' and at least in this pull request the argument here is not actually used.